frontend/public/components/network-policy.jsx

```diff
@@ -173,9 +174,10 @@ const IngressRow = ({ ingress, namespace, podSelector }) => {
   );
 };
 
-const Details_ = ({ obj: np }) => {
-  const networkPolicyDocs =
-    'https://kubernetes.io/docs/concepts/services-networking/network-policies/';
+const Details_ = ({ obj: np, flags }) => {
+  const networkPolicyDocs = flags[FLAGS.OPENSHIFT]
+    ? `${openshiftHelpBase}networking/configuring-networkpolicy.html'`
```

Comment:
We should consolidate the links in documentation.tsx like it was previously. See https://github.com/openshift/console/commit/2dae49a86c0d3a34c6ed969a575be3c54d47195a#diff-12887fc2371d38a701d3799cb2766141

That makes it easier to check if any of the links have been broken (since the full list is in one place) and avoids multiple references to the same topic in the code. We might be able to automate checking for broken links as well in CI.

Reply:
@spadgett - is the latest version more what you were thinking?  I kept it simple for now since we only have the one item in there instead of making a more complicated function.